Makefile

```diff
@@ -0,0 +1,64 @@
+LOCAL_IMAGE = acorn_docker:1.0
+REMOTE_IMAGE = merlinran/acorn_docker
+
+.PHONY: list
+list:
+	@echo "Welcome! You may want to try one of the following:\n---"; \
```

Comment:
`@` tells `make` to not prompt the command follows. The rest are just normal shell scripts. This rule is a trick to list the main targets from the `Makefile` itself.

Reply:
One caveat is to always use tab instead of whitespaces in the rule body, or `make` won't recognize it. Editor may warn you or correct for you if that happens.